front/src/views/recovery-password/RecoveryPassword.css

```diff
@@ -0,0 +1,24 @@
+.MyInput{
+    display: flex;
+    justify-content: center;
+}
+
+.col{
+    flex-grow: 0;
+}
+
+.v-card__actions{
+    padding: inherit;
+}
+
+.theme--light.v-stepper {
+    background: #F6F8FB;
+}
+
+.v-stepper__header {
+    box-shadow: none;
+}
+
+.v-stepper {
+    box-shadow: none;
+}
```

Comment:
I'm sure it is. the background is definitely only relevant for me, and on the box-shadow stepper it may not be useful for others, because with box-shadow it looks more beautiful, but I should not have this in my layout